hal/include/hal/ArduinoUnoAliases.h

```diff
@@ -0,0 +1,97 @@
+
+/** \addtogroup hal */
+/** @{*/
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_ARDUINO_UNO_H
+#define MBED_ARDUINO_UNO_H
+
+#if defined (TARGET_FF_ARDUINO) || (TARGET_FF_ARDUINO_UNO)
+
+#ifdef TARGET_FF_ARDUINO_UNO
+
+#define A0  ARDUINO_UNO_A0
+#define A1  ARDUINO_UNO_A1
+#define A2  ARDUINO_UNO_A2
+#define A3  ARDUINO_UNO_A3
+#define A4  ARDUINO_UNO_A4
+#define A5  ARDUINO_UNO_A5
+#define D0  ARDUINO_UNO_D0
+#define D1  ARDUINO_UNO_D1
+#define D2  ARDUINO_UNO_D2
+#define D3  ARDUINO_UNO_D3
+#define D4  ARDUINO_UNO_D4
+#define D5  ARDUINO_UNO_D5
+#define D6  ARDUINO_UNO_D6
+#define D7  ARDUINO_UNO_D7
+#define D8  ARDUINO_UNO_D8
+#define D9  ARDUINO_UNO_D9
+#define D10 ARDUINO_UNO_D10
+#define D11 ARDUINO_UNO_D11
+#define D12 ARDUINO_UNO_D12
+#define D13 ARDUINO_UNO_D13
+#define D14 ARDUINO_UNO_D14
+#define D15 ARDUINO_UNO_D15
+
+#endif // TARGET_FF_ARDUINO_UNO
+
+#ifdef TARGET_FF_ARDUINO
+
```

Comment:
Would this be the place to add a deprecation message in the future?

Reply:
Answer to myself: This is done here https://github.com/ARMmbed/mbed-os/pull/13919/files#diff-3bb19a969fee5b0e2248bf506e921599057f2c3486331d0dfb604fa7dfa053b4R29

Reply:
> Would this be the place to add a deprecation message in the future?

I would say yes